src/playback/tests/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ set(MODULE_TEST_LINK
 
     muse::uicomponents
     muse::ui
+
+    Qt::Quick
```

Comment:
Is it `playback_tests` that needs to link to Qt::Quick, or is it `playback`?

(Also, at MU, everything links to `global`, and `global` publicly links to `Qt::Quick` because of PCH. So at MU, something like this is not necessary.)

Reply:
> How does it affect compile time?

Hm, I haven't investigated that, sorry